scripts/b.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import pkgutil
+import re
+import sys
+
+# Don't make .pyc files
+sys.dont_write_bytecode = True
+
+scripts_dir = os.path.dirname(os.path.realpath(__file__))
+project_dir = os.path.dirname(scripts_dir)
+build_dir = os.path.join(project_dir, "build")
+
+# Build our cmake cache
+cmake_cache = {}
+
+dirs = [build_dir]
+try:
+    dirs.extend([os.path.join(build_dir, f) for f in os.listdir(build_dir)])
+except FileNotFoundError:
+    pass
+
+for d in dirs:
+    if os.path.isfile(os.path.join(project_dir, d, "CMakeCache.txt")):
```

Comment:
`d` should already be an absolute path
```suggestion
    if os.path.isfile(os.path.join(d, "CMakeCache.txt")):
```